tests/monitoring_test.go

```diff
@@ -92,40 +92,29 @@ var _ = Describe("Exporters", func() {
 	})
 
 	DescribeTable("Exported timeseries", func(selector string, match types.GomegaMatcher) {
+		var series []map[string]string
```

Comment:
I think this should be inside `Eventually(func() { ... })` too..

Reply:
done!